.gitignore

```diff
@@ -4,6 +4,7 @@
 /dist
 /tmp
 /out-tsc
+/api/apps
```

Comment:
why do we need this? what is /apps dir?

Reply:
it is a backend application, I don't think we need to commit it